src/NSwag.Sample.NETCore21/Controllers/PetController.cs

```diff
@@ -87,6 +91,7 @@ public class PetController : ControllerBase
         [HttpPost("{petId}")]
         [Consumes("application/www-form-url-encoded")]
         [Produces("application/json")]
+        [ProducesResponseType(typeof(void), (int)HttpStatusCode.NoContent)]
```

Comment:
You could do `ProducesResponseType((int)HttpStatusCode.NoContent)` and avoid specifying `typeof(void)`